include/uxr/client/core/session/create_entities_xml.h

```diff
@@ -30,7 +34,7 @@ extern "C"
  * @param session       A uxrSession structure previously initialized.
  * @param stream_id     The output stream identifier where the CREATE submessage will be buffered.
  * @param object_id     The identifier of the XRCE Participant.
- * @param domain_id     The identifier of the Domain to which the XRCE Participant belongs.
+ * @param domain        The identifier of the Domain to which the XRCE Participant belongs.
```

Comment:
why not `domain_id`?

Reply:
Because the @param name must match the function parameter name, which is `domain`:

https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/7c3e2f491191370e158809736cc3842b2754bb13/include/uxr/client/core/session/create_entities_xml.h#L45-L50

Reply:
Let's change all to `domain_id`